src/components/TermsandConditions/TermsAndConditionsContent.js

```diff
@@ -0,0 +1,228 @@
+/* eslint-disable max-len */
+import React from "react";
+
+const TermsAndConditionsContent = () => (
+    <div id="doc" style={{ fontFamily: "Poppins, Roboto, sans-serif" }}>
```

Comment:
do not use `style` directly. Use `makeStyles` and `className`

Reply:
Also, you don't need the style at all here, if you swap the `<p>` and `<h1>`, etc... with `<Typography>` from MUI, which will take the theme defaults (font and color), which you should

https://v4.mui.com/pt/components/typography/

Reply:
I'm working on it, is there any problem to use `&nbsp;`, instead of `span`?

Reply:
It's generally a bad practice to rely on structure elements (&nbsp;) to fix design issues (spacing, I assume), so only use it if extremely necessary